backend/web/index.php

```diff
@@ -1,7 +1,5 @@
 <?php
-defined('YII_DEBUG') or define('YII_DEBUG', true);
-defined('YII_ENV') or define('YII_ENV', 'dev');
-
+require(__DIR__ . '/../../common/config/environment.php');
```

Comment:
I can't see `common/config/environment.php`

Also this may be rise negative feedbeck and BC broken:

Any changes with environment require re-run the initi file. 

Again.

After composer update.